content/docs/cucumber/api.md

```diff
@@ -848,6 +848,23 @@ The **Command-Line Interface Runner (CLI Runner)** is an executable Java class t
 ```
 java io.cucumber.core.cli.Main
 ```
+Note that you will need to add the following jars to your classpath, in addition to the location of your compiled .class files:
+
+* `cucumber-core-{{% version "cucumberjvm" %}}.jar`
+* `gherkin-{{% version "gherkin" %}}.jar`
+* `cucumber-expressions-{{% version "cucumberjvm" %}}.jar`
+* `datatable-{{% version "datatable" %}}.jar`
+* `datatable-dependencies-{{% version "datatable" %}}.jar`
+* `cucumber-java-{{% version "cucumberjvm" %}}.jar`
+
+Be sure to use the same version for all cucumber-jvm dependencies.  You can find these jars in [Maven Central](https://mvnrepository.com/repos/central).
+You will also need to provide the CLI with your step definitions via the `--glue` option followed by its package name, and the location of your feature file(s).
```

Comment:
Let's also add how to provide the location of the feature files (same as for the glue)?

Reply:
There's no --feature option on the CLI as there is for glue code, so I modified the wording on this some (you just provide the file path to the feature files).  There is also an example command provided.